tomcat/src/main/java/com/linecorp/armeria/server/tomcat/TomcatServiceConfig.java

```diff
@@ -106,10 +106,11 @@ Path docBase() {
     /**
      * Returns the path to the root directory of a web application inside a JAR/WAR.
      *
-     * @return {@link Optional#empty()} if {@link #docBase()} is not a JAR/WAR file
+     * @return {@code null} if {@link #docBase()} is not a JAR/WAR file
```

Comment:
Here especially, I'd expect just `, or {@code null}`... inlined with the first line.